src/server.js

```diff
@@ -4,11 +4,16 @@ import { readdirSync } from 'fs'
 import http from 'http';
 import path from 'path';
 import config from './config.js';
+import { WebSocketServer } from 'ws';
+import OpenAI from 'openai';
 
 const app = express();
 const server = http.createServer(app);
 const componentsDir = path.join(process.cwd(), 'src/components');
-const { port } = config;
+const { port, openaiApiKey } = config;
+const wss = new WebSocketServer({ server });
+const sessions = new Map();
+const SYSTEM_PROMPT = 'You are a helpful assistant. This conversation is being translated to voice, so answer carefully. When you respond, please spell out all numbers, for example twenty not 20. Do not include emojis in your responses. Do not include bullet points, asterisks, or special symbols.';
```

Comment:
This is specific to ConversationRelay. It should live under `src/components/twilio-voice-conversation-relay/` folder.